lib/middleware/tagged_logging.rb

```diff
@@ -20,5 +22,14 @@ def call(env)
       ::Logging.mdc.delete('request')
       ::Logging.mdc.delete('session')
     end
```

Comment:
The request ID is already visible via the `proxy` logger, I'm not adding an additional INFO log entry to every request.